Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -60,13 +69,32 @@ ProjectPamphletViewModelOutputs {
 
     self.prefersStatusBarHiddenProperty <~ self.viewWillAppearAnimated.signal.mapConst(true)
 
-    self.setNeedsStatusBarAppearanceUpdate = self.viewWillAppearAnimated.signal.ignoreValues()
+    self.setNeedsStatusBarAppearanceUpdate = Signal.merge(
+      self.viewWillAppearAnimated.signal.ignoreValues(),
+      self.willTransitionToCollectionProperty.signal.ignoreValues()
+    )
 
     self.setNavigationBarHiddenAnimated = Signal.merge(
       self.viewDidLoadProperty.signal.mapConst((true, false)),
       self.viewWillAppearAnimated.signal.skip(first: 1).map { (true, $0) }
     )
 
+    var initialTopConstraint: CGFloat = 0.0
```

Comment:
I need to look into this a little closer but I think there's probably a different, more 'FRP', approach to capturing this local variable.